olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml.diff

```diff
@@ -0,0 +1,27 @@
+--- /home/ibuziuk/go/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594362070/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594362070.clusterserviceversion.yaml	2020-07-13 15:26:40.113184452 +0200
++++ /home/ibuziuk/go/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml	2020-07-13 15:35:59.597697532 +0200
+@@ -52,13 +52,14 @@
+     categories: Developer Tools
+     certified: "false"
+     containerImage: quay.io/eclipse/che-operator:nightly
+-    createdAt: "2020-07-10T06:21:10Z"
++    createdAt: "2020-07-13T13:35:59Z"
+     description: A Kube-native development solution that delivers portable and collaborative
+       developer workspaces.
+     operatorframework.io/suggested-namespace: eclipse-che
++    operatorframework.io/cluster-monitoring: "true"
```

Comment:
Does it make sense for raw, non-OpenShift Kubernetes installations ?

Reply:
@davidfestal technically, it should be possible to install on OpenShift via the kubernetes csv, right?
In general, this property would make sense only for CRW, but I thought that CRW csv will fall back on upsteam, no?

Reply:
> it should be possible to install on OpenShift via the kubernetes csv, right?

well, that's not recommended at all. Not even sure it works.

> CRW csv will fall back on upsteam

Not sure what you mean here ? CRW CSV is based on the upstream OpenShift CSV

Are we mixing *upstream* Che and *upstream* Kubernetes here ?